worlds/candybox2/export_rules_package.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+if __name__ == "__main__":
+    from worlds.candybox2.rules import CandyBox2RulesPackage, generate_rules_package
```

Comment:
```suggestion
    from .rules import CandyBox2RulesPackage, generate_rules_package
```
I guess this one doesn't _really_ need relative imports since you only use this for dev stuff, but might as well

Reply:
This change was causing import errors when running the script, but that might be me having something misconfigured on my end. Will have to look into this more